doc/data/messages/i/invalid-all-object/bad.py

```diff
@@ -0,0 +1,7 @@
+class Fruit:
+    pass
+
+class Worm:
+    pass
+
+__all__ = [Fruit, Worm]  # [invalid-all-object]
```

Comment:
`__all__` is supposed to be at the top of the file

Reply:
Related information has been added to `details.rst` and `related.rst` file. And examples have been revised.